base/dsfs/package.go

```diff
@@ -53,8 +53,10 @@ const (
 	PackageFileViz
 	// PackageFileRenderedViz is the rendered visualization of the dataset
 	PackageFileRenderedViz
-	// PackageFileReadme is the raw readme of the dataset
+	// PackageFileReadme connects readme data to the dataset package
```

Comment:
Should this be a hidden file like `.qri-ref` if it's only a link?

Reply:
This comment is somewhat misleading, it's more than a link, it also specifies the syntax ("markdown") readme is encoded with.